curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/6567055f59d39f07d1c542dc.md

```diff
@@ -0,0 +1,634 @@
+---
+id: 6567055f59d39f07d1c542dc
+title: Step 18
+challengeType: 0
+dashedName: step-18
+---
+
+# --description--
+
+Next, you will need to update the playlist in your HTML document to display the songs.
+
+Use `innerHTML` property to add `songsHTML` to the `playlistSongs`.
```

Comment:
```suggestion
Use the `innerHTML` property to add `songsHTML` to the `playlistSongs`. This will insert the `li` element you just created into the `ul` element in the already provided `HTML` file.```